contracts/src/node-interface/NodeInterface.sol

```diff
@@ -64,4 +64,31 @@ interface NodeInterface {
      * @return confirmations The number of L1 confirmations the sequencer batch has. Returns 0 if block not yet included in an L1 batch.
      */
     function getL1Confirmations(bytes32 blockHash) external view returns (uint64 confirmations);
+
+    /**
+     * @notice Same as native gas estimation, but with additional info on the l1 costs
+     * @param ignoreTo whether to drop the "to" field
```

Comment:
would be cool to clarify this comment
```suggestion
     * @param ignoreTo whether to drop the "to" field. This is needed for estimating contract deployments.
```

Reply:
I've renamed `ignoreTo` to `contractCreation` and added an explanation that this ignores the `To` field